ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -663,23 +665,26 @@ - (void)_setBackgroundColor:(UIView *)view
 {
     NSString *backgroundColorString = [self _readBackgroundColorFromManifest:_appRecord.appLoader.manifest];
     UIColor *backgroundColor = [EXUtil colorWithHexString:backgroundColorString];
+    view.backgroundColor = [UIColor clearColor];
 
     if (backgroundColor) {
-      view.backgroundColor = backgroundColor;
-      // NOTE(brentvatne): it may be desirable at some point to split the window backgroundColor out from the
-      // root view, we can do if use case is presented to us.
+      if (view.window.rootViewController != nil && view.window.rootViewController.view != nil) {
```

Comment:
maybe to add some comment about what's the `view.window.rootViewControlller` expected to be and why we should mutate its backgroundColor.

and refactor `_setBackgroundColor:` where `[self _setBackgroundColor:self.view];` is redundant.  
e.g.

```objc
- (void)_setBackgroundColor
{
  NSString *backgroundColorString = [self _readBackgroundColorFromManifest:_appRecord.appLoader.manifest];
  UIColor *backgroundColor = [EXUtil colorWithHexString:backgroundColorString];

  if (self.view.window.rootViewController != nil && self.view.window.rootViewController.view != nil) {
    self.view.window.rootViewController.view.backgroundColor = backgroundColor ? backgroundColor : [UIColor whiteColor];
  }

  self.view.window.backgroundColor = backgroundColor;
}
```